description: remove some prune causing build to fail, modify GN flags
author: Wengling Chen <feiyu2817@gmail.com>


--- a/ungoogled-chromium/pruning.list
+++ b/ungoogled-chromium/pruning.list
@@ -17,7 +17,6 @@ base/test/data/pe_image/pe_image_test_32.dll
 base/test/data/tzdata/2019a/44/le/timezoneTypes.res
 base/test/data/tzdata/2019a/44/le/zoneinfo64.res
 build/android/CheckInstallApk-debug.apk
-build/android/chromium-debug.keystore
 build/android/stacktrace/java_deobfuscate.jar
 build/android/tests/symbolize/liba.so
 build/android/tests/symbolize/libb.so
@@ -14727,12 +14725,9 @@ third_party/hunspell/fuzz/bdict_corpus/en-US-7-1.bdic
 third_party/hunspell/fuzz/bdict_corpus/en-US-8-0.bdic
 third_party/hunspell/fuzz/bdict_corpus/f95048cdc0fc231dd712bad106578a9eeecd869b
 third_party/hunspell/fuzz/bdict_corpus/fda69541054125551c41ab3e591e0482405ca2dc
-third_party/icu/android/icudtl.dat
-third_party/icu/android_small/icudtl.dat
 third_party/icu/android_small/icudtl_extra.dat
 third_party/icu/cast/icudtl.dat
 third_party/icu/chromeos/icudtl.dat
-third_party/icu/common/icudtb.dat
 third_party/icu/flutter/icudtl.dat
 third_party/icu/ios/icudtl.dat
 third_party/icu/source/data/in/coll/ucadata-implicithan.icu
@@ -15178,7 +15178,6 @@ base/test/data/pe_image/pe_image_test_32.dll
 third_party/icu/tzres/metaZones.res
 third_party/icu/tzres/timezoneTypes.res
 third_party/icu/tzres/zoneinfo64.res
-third_party/jetifier_standalone/lib/jetifier-standalone.jar
 third_party/libdrm/src/intel/tests/gen4-3d.batch
 third_party/libdrm/src/intel/tests/gen5-3d.batch
 third_party/libdrm/src/intel/tests/gen6-3d.batch
--- a/ungoogled-chromium/flags.gn
+++ b/ungoogled-chromium/flags.gn
@@ -11,7 +11,6 @@
 enable_reporting=false
 enable_service_discovery=false
 enable_widevine=true
-exclude_unwind_tables=true
 fieldtrial_testing_like_official_build=true
 google_api_key=""
 google_default_client_id=""
